lib/cli/src/Cardano/CLI.hs

```diff
@@ -1538,13 +1557,39 @@ initTracer
 initTracer loggerName outputs = do
     cfg <- do
         c <- defaultConfigStdout
-        CM.setSetupBackends c [CM.KatipBK, CM.AggregationBK]
+        CM.setSetupBackends c [CM.KatipBK, CM.AggregationBK, CM.EKGViewBK, CM.EditorBK]
+        CM.setDefaultBackends c [CM.KatipBK, CM.EKGViewBK]
         CM.setSetupScribes c $ map mkScribe outputs
         CM.setDefaultScribes c $ map mkScribeId outputs
+        CM.setEKGBindAddr c $ Just (Endpoint ("127.0.0.1", 13788))
```

Comment:
I guess we want the ports be configurable... or alternatively, allow all of the logging config configurable and use yaml file like node does?

Reply:
Yeah, we should probably add some CLI argument(s) in the style of
```
  --random-port            serve wallet API on any available port (conflicts
                           with --port)
  --port INT               port used for serving the wallet API. (default: 8090)
```

Reply:
I think it's a rare enough use case to hide it behind env vars, which I just did. The test cluster than sets those env vars.

Reply:
> I think it's a rare enough use case to hide it behind env vars, which I just did.

Sounds sane 👍 

But: wouldn't it be nice if we extended the trace of the wallet url and port, to also include the EKG url/port (and maybe Prometheus) https://github.com/input-output-hk/cardano-wallet/blob/0c419518edf98cdb5f701fd6c65fe6832a89afad/lib/shelley/test/integration/Main.hs#L303 Otherwise it would be tricky to check the EKG of the integration tests wallet.

Reply:
Done